### DISCUSSION TITLE
Can't verify academic affiliation

### DISCUSSION THREAD

**Discussion by duoLips** [community/community#139132] [GitHub Education]
### Hi everyone,

Seems like no matter what proof I provide, gitHub won't approve it. 

Took a terrible quality pictures(since only front camera works) of my student id, university issued note about affiliation, schedule that shows all classes are online, official document from Ministry of Education, stating that during war all education is remote. I also added legal name to my github account.

Never had issues with getting educational packs before. Had autodesk and jetbrains(before it switched to this monstrosity) approve everything in less than a week.
Is there anything else I can do?

**Comment by alx-mp4** [comment]
Hi @duoLips, I recommend opening a ticket with GitHub support for this issue. I had applied for the GitHub Education Program myself, but my application was declined. If your photos are blurry, contain a foreign language, or are missing required information, the application is likely to be rejected. Make sure your photos meet all the requirements, and if they’re not in English, include a translation before resubmitting.

If this has helped you, I would appreciate if my answer is marked as an answer. Thank you!
